src/Presenters/Content/WikiContentProvider.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend\Presenters\Content;
```

Comment:
Will do that. Does PageContentModifier belong there also?

2016-01-25 14:29 GMT+01:00 Jeroen De Dauw <notifications@github.com>:

> In src/Presenters/Content/WikiContentProvider.php
> <https://github.com/wmde/FundraisingFrontend/pull/86#discussion_r50692784>
> :
>
> > @@ -0,0 +1,40 @@
> > +<?php
> > +
> > +namespace WMDE\Fundraising\Frontend\Presenters\Content;
>
> Think WMDE\Fundraising\Frontend\DataAccess is more appropriate
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wmde/FundraisingFrontend/pull/86/files#r50692784>.
>



-- 
Gabriel Birke
Software Developer

Wikimedia Deutschland e. V. | Tempelhofer Ufer 23-24 | 10963 Berlin
Phone: +49 (0)30 219 158 26-0
http://wikimedia.de

Imagine a world, in which every single human being can freely share in the
sum of all knowledge. That‘s our commitment.

Wikimedia Deutschland - Gesellschaft zur Förderung Freien Wissens e. V.
Eingetragen im Vereinsregister des Amtsgerichts Berlin-Charlottenburg unter
der Nummer 23855 B. Als gemeinnützig anerkannt durch das Finanzamt für
Körperschaften I Berlin, Steuernummer 27/681/51985.